frontend/src/views/Homepage/index.tsx

```diff
@@ -1,8 +1,29 @@
+import loadable from "@loadable/component";
 import React, { FC } from "react";
+import { get } from "src/common/featureFlags";
+import { FEATURES } from "src/common/featureFlags/features";
+import { BOOLEAN } from "src/common/localStorage/set";
 import Collections from "src/components/Collections";
 
+const AsyncUploadCSV = loadable(
```

Comment:
Code split `UploadCSV` component to only load if `?csv=true` in URL, since the component is very heavy!